src/deploy_inst.h

```diff
@@ -49,18 +49,18 @@ class DeployInst : public cyclus::Institution {
   std::vector<std::string> prototypes;
 
   #pragma cyclus var { \
-    "doc": "Time step on which to deploy agents given in prototype list (same order).", \
-    "uilabel": "Deployment times", \
-  }
-  std::vector<int> build_times;
-
-  #pragma cyclus var { \
     "doc": "Number of each prototype given in prototype list that should be deployed (same order).", \
     "uilabel": "Number to deploy", \
   }
   std::vector<int> n_build;
 
   #pragma cyclus var { \
+    "doc": "Time step on which to deploy agents given in prototype list (same order).", \
+    "uilabel": "Deployment times", \
+  }
+  std::vector<int> build_times;
```

Comment:
This ordering can probably go both ways, but I have always though of it the other way.  First choose a time, and then choose how many to build then.  From a key=value perspective, the build time is the unique variable.